Gordon360/Startup.cs

```diff
@@ -170,7 +175,6 @@ private void DoMyWork()
             {
                 JObject theFacStaff = JObject.FromObject(aFacStaff);
                 theFacStaff.Add("Hall", null);
-                theFacStaff.Add("Mail_Location", null);
```

Comment:
Yes, because Mail_Location is now intentionally included on FacStaff profiles per request of the Mail Room. You can see those updates in the relevent ViewModels in this pr.